### DISCUSSION TITLE
Remove titlebar from window

### DISCUSSION THREAD

**Discussion by Jimmeywho** [TomSchimansky/CustomTkinter#1005] [Q&A] [answered]
I am building an application where I would like to remove the standard title bar of the window and utilise an exit button within.
I have tried a suggested route of wm_attributes with type as splash, which was identified from standard tkinter, however it does not appear to be built into customtkinter and errors.
Has anyone else produced a window like this, and if so, how did you achieve it?
Thank you in advance!

**Comment by Akascape** [comment, marked as answer]
@Jimmeywho Yes, I have created a proper window removing the standard title bar in windows. 
Here is the ready-to-use template:
https://github.com/TomSchimansky/CustomTkinter/discussions/736

You can further customize it easily.

**Reply by Jimmeywho** [reply]
Superstar! Thank you!
That is exactly what I was looking for and more. Answered some of nice to have queries I had for the future too!
